NOTICE

```diff
@@ -14,6 +14,10 @@ Mission Support System (MSS)
    reference documentation in any publication, presentation, report,
    etc. you create:
 
+   Bauer, R., Grooß, J.-U., Ungermann, J., Bär, M., Geldenhuys, M., and Hoffmann, L.: The Mission Support
+   System (MSS v7.0.4) and its use in planning for the SouthTRAC aircraft campaign, Geosci.
+   Model Dev., 15, 8983–8997, https://doi.org/10.5194/gmd-15-8983-2022, 2022.
+
    Rautenhaus, M., Bauer, G., and Doernbrack, A.: A web service based
    tool to plan atmospheric research flights, Geosci. Model Dev., 5,
    55-71, doi:10.5194/gmd-5-55-2012, 2012.
```

Comment:
and here one more.

that may be a vscode issue not sure, never used that myself. In pycharm I would have found all occurencies by a search.

Reply:
> The NOTICE file also has the old URL regarding 2012 paper, shall I update that too?
Yeah, I saw that, and I resolved it.